tests/unit/FutureEffective.spec.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint max-len: 0 */
```

Comment:
You'll notice I verify the entire lines of text below. I can't think of a reason not to.

I formatted them as below for easier readability.